packages/rum/src/rum.entry.ts

```diff
@@ -92,13 +92,15 @@ export function makeRumGlobal(startRumImpl: StartRum) {
     }),
 
     addUserAction: monitor((name: string, context?: Context) => {
-      addUserActionStrategy({
-        name,
-        context: combine({}, context),
-        globalContext: combine({}, globalContextManager.get()),
-        startTime: performance.now(),
-        type: UserActionType.CUSTOM,
-      })
+      addUserActionStrategy(
+        {
+          name,
+          context: combine({}, context),
+          startTime: performance.now(),
+          type: UserActionType.CUSTOM,
+        },
+        combine({}, globalContextManager.get())
```

Comment:
as asked in https://github.com/DataDog/browser-sdk/pull/551#discussion_r501709021, do we need to copy the context in after init?

if not, what about passing global context only on before init and use the current global context as usual on after init?

I have a hard time legitimate the test `it('should ignore the current global context', () => {`

Reply:
You are right, we don't need to clone the global context after init. I can move this clone in the 'before init' `addUserActionStrategy`, everything should be fine.